app/assets/javascripts/discourse/app/components/ace-editor.gjs

```diff
@@ -143,6 +143,9 @@ export default class AceEditor extends Component {
     this.editor.$blockScrolling = Infinity;
     this.editor.renderer.setScrollMargin(10, 10);
 
+    // For testing purposes
+    element.aceEditor = this.editor;
```

Comment:
Worth wrapping in `if(isTesting())`, to make sure themes/plugins don't use it in prod?